src/main/java/org/embulk/output/sf_bulk_api/SForceColumnVisitor.java

```diff
@@ -42,7 +42,8 @@ public void longColumn(Column column) {
     if (pageReader.isNull(column)) {
       addFieldsToNull(column);
     } else {
-      record.addField(column.getName(), Long.toString(pageReader.getLong(column)));
+      // number type appears to be double type
+      record.addField(column.getName(), (double) pageReader.getLong(column));
```

Comment:
Is this the correct type converting?
```suggestion
      record.addField(column.getName(), (long) pageReader.getLong(column));
```

Reply:
I understand this is not natural, 
but long type does not work appropriately when this field is index.

Reply:
I added comments https://github.com/trocco-io/embulk-output-sf_bulk_api/pull/14/commits/517e98339c06d8f65507720731f0a3631e0475cc